assets/src/edit-story/elements/video/controls.js

```diff
@@ -255,7 +260,7 @@ function VideoControls({
             onMouseDown={handlePlayPause}
             isAbove={isPlayAbove}
           >
-            {isPlaying ? <Pause /> : <Play />}
+            <Icon isRTL={isRTL} />
```

Comment:
Should this be a transient prop or isn't this a problem?

Reply:
`isRTL` is not a problem.